code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -470,7 +470,7 @@
 			if("gibbfloats")
 				icon_state = "gibbfloats"
 				name = "Gibbfloat"
-				desc = "Dr. Gibb with ice cream on top."
```

Comment:
This isn't actually a grammar error. Ice Cream is actually the correct way to write it.